highfidelity/hifi
# CR 21383: Integrate attachments into doppleganger mirror

#### Testing:
* Launch the new doppleganger / mirror script from this PR; rawgit link http://cdn.rawgit.com/humbletim/hifi/d6eb77e/unpublishedScripts/marketplace/doppleganger-attachments/app-doppleganger-attachments.js
* Make sure to be wearing the default Mannequin avatar.
* Verify the new tablet/toolbar button appears.
* Activate the doppleganger mirror.
* Try adding/removing/adjusting various attachments:
    * ... @ welcome (or dev-welcome) shops.
    * ... manually using the Avatar > Attachments UI dialog.
* Verify each attachment change is reflected on the doppleganger.
